packages/cli/src/lib/config.ts

```diff
@@ -38,6 +39,14 @@ async function createConfig(): Promise<Config> {
 }
 
 export async function loadOrCreateConfig(path = DEFAULT_CONFIG_PATH): Promise<Config> {
+  if (!existsSync(path)) {
+    // no new named config file found
+    if (existsSync(join(homedir(), '/.llama-pack/llama-pack-config.json'))) {
```

Comment:
It would be nice to log something to the console here, just to let the user know what's happening.